tests/ci/docker_images/build_cryptofuzz_modules.sh

```diff
@@ -36,11 +36,8 @@ make -j$(nproc)
 
 # Crypto++ https://github.com/guidovranken/cryptofuzz/blob/master/docs/cryptopp.md
 cd "$MODULES_ROOT"
-# Crypto++ broke arm compatibility with this change https://github.com/weidai11/cryptopp/commit/d1cea852f06008541966de7af433b57c4e9e2504
```

Comment:
So this pinning didn't fix the Arm build?

Reply:
Picking the other version of Crypto++ fixed this issue. They also fixed the issue with the latest commit so pinning isn't necessary. I still need to turn the ARM build off because something else is breaking in Cryptofuzz/cpu_features.